ddtrace/tracer/span.go

```diff
@@ -99,7 +99,10 @@ func (s *span) SetTag(key string, value interface{}) {
 
 // setTagError sets the error tag. It accounts for various valid scenarios.
 // This method is not safe for concurrent use.
-func (s *span) setTagError(value interface{}) {
+func (s *span) setTagError(value interface{}, backtrace bool) {
+	if s.finished {
```

Comment:
Do we need this? It's already applied in `SetTag` and this one's internal. Let's remove it.

Reply:
Yes, it's needed because `setTagError` is called directly in `Finish`. I assume we don't want to override the original error if `Finish` called again with different error.

Reply:
SGTM